packages/unenv-preset/tests/worker/index.ts

```diff
@@ -166,3 +169,10 @@ export async function testNet() {
 	assert.strictEqual(typeof net.createConnection, "function");
 	assert.throws(() => net.createServer(), /not implemented/);
 }
+
+export async function testTls() {
+	const tls = await import("node:tls");
+	assert.strictEqual(typeof tls, "object");
+	// @ts-expect-error Invalid @types/node definition.
```

Comment:
could/should we open an issue in DefinitelyTyped to fix the type definition on their end?

(of course it's non-blocking comment 🙂)

Reply:
Awesome, thanks :smiley: